mava/utils/logger_tools.py

```diff
@@ -249,17 +96,45 @@ def get_logger() -> logging.Logger:
     return logger
 
 
-def config_copy(config: Dict) -> Dict:
-    """Deep copy a config."""
-    if isinstance(config, dict):
-        return {k: config_copy(v) for k, v in config.items()}
-    elif isinstance(config, list):
-        return [config_copy(v) for v in config]
-    else:
-        return deepcopy(config)
+def get_neptune_logger(cfg: Dict) -> neptune.Run:
+    """Set up neptune logging."""
+    name = cfg["name"]
+    tags = cfg["neptune_tag"]
+    project = cfg["neptune_project"]
```

Comment:
If you can add this to config `neptune_project: "InstaDeep/Mava"` else try to set it to the default value.